### ISSUE TITLE
Strength 3 Blocks are causing Void Outs

### ISSUE THREAD

**Issue by r0bd0g** [TestRunnerSRL/OoT-Randomizer#1054]
https://clips.twitch.tv/OpenMagnificentChimpanzeeAsianGlow

Suspected to be caused by the recent speedup.

**Comment by Roman971** [comment]
Just to be clear, pickup speedups are not in this fork yet.

**Comment by r0bd0g** [comment]
Ah, I thought it was :p. Okay Roman fork only.

**Comment by fenhl** [comment]
Was this speedup ever merged into this fork?

**Comment by fenhl** [comment]
I don't see anything that looks related in the Dev-R diff, and testing on latest Dev I don't see any timing differences compared to the video. So I have to assume that this speedup is in main Dev now. But I've also never seen this happen to anyone else so either it's already fixed or it requires extremely specific circumstances to reproduce.

**Comment by r0bd0g** [comment]
I think it's happened like twice.
I dunno what causes it, so I make a point not to mess around when I go to pick up these blocks. That and the fact that most people aren't lifting these blocks very often, you probably don't see it much. I'll mash buttons next time I'm lifting these I guess.

**Comment by djevangelia** [comment]
I tried it out like in the clip (MQ Fire trial, Biggorons out) and nothing happened.

But to me the far more interesting thing in the clip is that the grab is not perpendicular to the block but at an angle. In vanilla (and main dev branch) it is not possible to grab from an angle, because the code that initiates heavy rock grab (Player_ActionHandler_5) will put the player perpendicular to the block when calling the setup function, no matter what. I could only recreate this in vanilla by removing that line of code. https://github.com/djevangelia/oot/blob/bc8a88fec80683a0849ae01afd168eba7a883a66/src/overlays/actors/ovl_player_actor/z_player.c#L8728

So in the branch in the clip there must have been other things changed in the heavy block handling. If those heavy block handling changes are not present in main dev branch and no one else has experienced it in main dev, it might not be a bug of the main dev branch.

**Comment by fenhl** [comment]
Agreed. It seems most likely that this is either fixed or was never an issue here, so I'm closing this. If anyone is still seeing this on current main Dev, please leave a comment.

**Comment by r0bd0g** [comment]
I wish we knew for sure that there is a difference in the heavy block code between main and this old roman fork. I feel like I've seen a sideways grab in the last year or two... I probably wasn't playing on main, but it seems plausible to me that other side branches could still have this issue idk.
